locale/es/manageiq.po

```diff
@@ -424,9 +424,9 @@ msgstr[1] ""
 
 msgid "%{amount} Automation Template Being Tagged"
 msgstr ""
-"{n, plural, other {%'{'amount'}' Automation Templates Being Tagged} one {%' {'"
-"amount'} ' Plantilla de automatización que se está etiquetando} many {%'{'"
-"amount'}' Automation Templates Being Tagged}}"
+"{n, plural, other {%{amount} Automation Templates Being Tagged} one "
+"{%{amount} Plantilla de automatización que se está etiquetando} many "
+"{%{amount} Automation Templates Being Tagged}}"
```

Comment:
We should change this at the source to allow for multiples (so it would have the msgid_plural

Reply:
I reverted this and put what we got for french and german... untranslated singular and plural forms.  Will review this with translators after merge and try to get it fixed.